.github/workflows/ci.yml

```diff
@@ -1,6 +1,10 @@
 
 name: "build"
-on: push
+on:
+  push:
+    branches:
+      - master
```

Comment:
Why only `master`? Surely we want CI to run on our branches too?

I recommend
```
on: [push, pull_request]
```

Reply:
I tried this before. It will trigger two CI jobs if you are pushing PRs directly in the repo.  `pull_request` is actually enough for running CIs for all branches. The `push on master` is there to make sure we don't break master after the merge.